bin/cli

```diff
@@ -9,14 +9,15 @@ if (!file_exists($autoloadPath)) {
     exit(1);
 }
 
-set_error_handler(function ($errno, $errstr, $errfile, $errline, array $errcontext): bool {
-    if (!(error_reporting() & $errno)) {
-        // respect error_reporting() level
-        // libraries used in custom components may emit notices that cannot be fixed
-        return false;
+set_error_handler('exceptions_error_handler');
```

Comment:
Tohle mi nepřijde správně. Zbytečně na tom nefunguje refactoring, pokud někdy někdo přejmenuje tu funkci. Proč je to vlastně změněné?

Reply:
jo jasne mas pravdu. Wtf. tady jsem nejak ujel, ta puvodni verze byla ok, opravim.